src/pages/products/turn/connecting.astro

```diff
@@ -0,0 +1,181 @@
+---
+import type { IStandardPageMetadata } from '@models/products/sgid/types';
+
+import QuickLinks from '@components/page/QuickLinks.astro';
+import Section from '@components/page/Section.astro';
+import Layout from '@layouts/FullWidth.astro';
+
+import { stripeSections } from '@utils/page';
+
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
+const page: IStandardPageMetadata = {
+  title: 'Getting Connected',
+  pillar: 'TURN',
+  section: stripeSections([
+    {},
+    { title: 'Getting Connected' },
+    { title: 'TURN Network' },
+    { title: 'Nevada Network' },
+    { title: 'Support Contacts' },
+  ]),
+};
+---
+
+<Layout title={page.title} pillar={page.pillar}>
+  <QuickLinks links={page.section} slot="quick-links" />
+
+  <Section {...page.section[1]}>
+    <p>
+      To create a new subscription or renew an existing subscription for Nevada GPS, you'll need to provide the
+      following information:
+    </p>
+    <ul>
```

Comment:
**polish**: @DenisePeterson import the BulletedList element, replace all the `<ul>`'s and put the `<li>`'s inside and remove the dot thing you added.

Here's an [example](https://github.com/agrc/gis.utah.gov/blob/main/src/pages/documentation/policy/data-sharing-agreement.astro#L88)